react-app/src/features/forms/waiver/app-k/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { describe, test, expect, beforeAll } from "vitest";
+import { skipCleanup, mockApiRefinements } from "@/utils/test-helpers/skipCleanup";
+import { renderForm } from "@/utils/test-helpers/renderForm";
+import { formSchemas } from "@/formSchemas";
+import { uploadFiles } from "@/utils/test-helpers/uploadFiles";
+import { AppKAmendmentForm } from ".";
+const upload = uploadFiles<(typeof formSchemas)["app-k"]>();
+
+describe("Appendix K", () => {
```

Comment:
I kinda want to insist on maintaining the capital letters for the test descriptions, but it's a losing battle at this point